actpool/queueworker.go

```diff
@@ -113,14 +115,27 @@ func (worker *queueWorker) Handle(job workerJob) error {
 
 	worker.mu.Lock()
 	defer worker.mu.Unlock()
+	if replace {
+		// TODO: early return if sender is the account to pop and nonce is larger than largest in the queue
+		actToReplace := worker.accountActs.PopPeek()
+		if actToReplace == nil {
+			log.L().Warn("UNEXPECTED ERROR: action pool is full, but no action to drop")
+			return nil
+		}
+		worker.ap.removeInvalidActs([]action.SealedEnvelope{*actToReplace})
+		if actToReplace.SenderAddress().String() == sender && actToReplace.Nonce() == nonce {
+			err = action.ErrTxPoolOverflow
```

Comment:
add a metric here, to count the number of this type error
`_actpoolMtc.WithLabelValues("overMaxNumActsPerPool").Inc()`